app/main.py

```diff
@@ -1,2 +1,118 @@
+from __future__ import annotations
+from dataclasses import dataclass
+from typing import Hashable, Any, Iterable
+
+INITIAL_CAPACITY = 8
+RESIZE_THRESHOLD = 2 / 3
+CAPACITY_MULTIPLIER = 2
+
+
+@dataclass
```

Comment:
This class better be defined inside of a Dictionary since it is a part of its implementation and will not be used outside of it
Also you should include hash_value as per task requirements